liota/dccs/iotcc.py

```diff
@@ -59,7 +59,7 @@ class IotControlCenter(DataCenterComponent):
 
     def __init__(self, con):
         log.info("Logging into DCC")
-        self._version = 20171023
+        self._version = 20171118
```

Comment:
We`ll have to change the version mechanism (probably later) keeping dates as version number is confusing as this date signifies the code review raised but the code will get checked-in lot later.

Reply:
We can discuss this offline in future for future CL.